src/components/KPagination/PaginationOffset.vue

```diff
@@ -7,7 +7,7 @@
       data-testid="previous-button"
       :disabled="previousButtonDisabled"
       icon
-      :size="isWideScreen ? 'medium' : 'small'"
+      size="small"
```

Comment:
I'm thinking we should do the inverse: default to medium and apply the small styles on the smaller viewport? 

The most common use-case for all of Konnect is always going to be Desktop, so having the classes match the rendered DOM seems more "normal" with `medium` as the default.

Reply:
That would mean we'd take desktop-first approach. I always thought we want to stick to mobile-first approach, across the board?

Reply:
The styles can be mobile-first, but the class reflected in the DOM would be un-manipulated

Reply:
I'd like to stick to this approach because reversing it would require writing a desktop-first query too (`@media (max-width: $kui-breakpoint-mobile)` instead of more traditional and preferable IMO, `min-width`).

Reply:
Couldn't the `medium` button appearance still be used, but your style rules would still apply?

Reply:
No. It's `small` by default and on screen size above mobile I apply `medium` size styles. If we made it `medium` by default I would need to apply `small` size styling on screen < mobile.

Reply:
What I'm worried about is someone trying to target `medium` buttons for padding via customization. I guess since we have unique classes on these buttons it's fine; just don't love the `button.small/button.medium` still there but 🤷🏼

Reply:
Yes, I think if someone is using `medium` or `small` classes for customization that's a bad approach in the first place but also chances of us doing those overrides anywhere in Konnect are very low.